EventFilter/CTPPSRawToDigi/src/RPixErrorChecker.cc

```diff
@@ -73,57 +93,111 @@ bool RPixErrorChecker::checkTrailer(bool& errorsInEvent, int fedId, unsigned int
   if ( fedTrailer.fragmentLength()!= nWords) {
     LogError("FedTrailerLenght")<< "fedTrailer.fragmentLength()!= nWords !! Fed: " << fedId << ", errorType = 34";
     errorsInEvent = true;
- 
+    if(includeErrors) {
+      int errorType = 34;
+      CTPPSPixelDataError error(*trailer, errorType, fedId);
+      errors[dummyDetId].push_back(error);
+    }
   }
   return fedTrailer.moreTrailers();
 }
 
-bool RPixErrorChecker::checkROC(bool& errorsInEvent, int fedId,  Word32& errorWord) const
+bool RPixErrorChecker::checkROC(bool& errorsInEvent, int fedId, uint32_t iD, Word32& errorWord,  Errors& errors) const
```

Comment:
just checking the past logic for the interface: why is the errorWord passed as non-const? It's not modified here

Reply:
right. done.